src/App.css

```diff
@@ -1,38 +1,40 @@
-.App {
-  text-align: center;
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.item-container {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  padding: 10px;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+.checkbox-item {
+  -webkit-transform: scale(2);
```

Comment:
since transform is not an experimental feature of css, we can drop the `-webkit` prefix. Check [this website](http://shouldiprefix.com/#transforms) for more info. In there, basically says that `-webkit` for transform was mandatory for older version of chrome, safari, etc.